src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -205,6 +205,15 @@ public async Task GetContentAsync_ErrorStatusCode_ExpectedExceptionThrown(bool w
             }
         }
 
+        [Fact]
+        [OuterLoop("Slow - Negative connection test")]
```

Comment:
why this would be slow if it connects to local host? that should fail quickly, right?

Reply:
Connection failure takes several seconds on Windows for some reason, even with localhost.

Reply:
how much delay do you see? The tests I was running were still pretty quick. I'm fine with leaving this outerloop.

Reply:
Failing bare Socket connections attempts take almost 2 seconds usually. This test runs even longer for some reason (about  4 seconds on my PC).